package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "simpleshop-cz",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "wp-scripts build --config webpack.config.js",
+    "start": "wp-scripts start --config webpack.config.js"
```

Comment:
Příkaz `npm run start` slouží pro build v průběhu vývoje. Sleduje to změny provedené v souborech a automaticky vybuildí novou verzi.